lib/webauto/web4cucumber.rb

```diff
@@ -147,9 +147,7 @@ def browser
         if Integer === @scroll_strategy
           chrome_caps[:element_scroll_behavior] = @scroll_strategy
         end
-        if self.class.container?
```

Comment:
Would you mind to try locally directly ( not in container ) ?

Reply:
@liangxia reverted these changes, PTAL